packages/video_player/tizen/src/video_player.cc

```diff
@@ -8,10 +8,21 @@
 #include <flutter/standard_method_codec.h>
 
 #include <algorithm>
+#include <queue>
 
 #include "log.h"
 #include "video_player_error.h"
 
+constexpr int kMessageQuit = -1;
+constexpr int kMessageOnFrameDecoded = 0;
+constexpr int kMessageOnRenderFinished = 1;
```

Comment:
This is okay for now but you may consider creating a dedicated enum type for constants like this in future implementations. (I think you just followed the current `TizenVsyncWaiter` implementation.)

Reply:
I have created message event enum.